go.mod

```diff
@@ -12,7 +12,8 @@ require (
 	github.com/gardener/dependency-watchdog v0.6.1-0.20210623112844-96f73d5dc311
 	github.com/gardener/etcd-druid v0.5.0
 	github.com/gardener/external-dns-management v0.7.18
-	github.com/gardener/gardener-resource-manager v0.18.0
+	github.com/gardener/gardener-resource-manager v0.25.0
+	github.com/gardener/gardener-resource-manager/api v0.0.0-00010101000000-000000000000
```

Comment:
😞 
Never liked that api module approach. It's confusing for IDEs, developers and now also to components that want to vendor it. So why are we still doing it, what's the benefit? Can we go back to the old variant?